internal/service/batch/job_definition_test.go

```diff
@@ -2144,3 +2206,41 @@ resource "aws_batch_job_definition" "test" {
 }
 `, rName, acctest.Region())
 }
+
+func testAccJobDefinitionConfig_simpleWithTags(rName string) string {
```

Comment:
The two config functions could be combined into a single function that takes a second parameter. I would clarify the intent and make the change more obvious.

Reply:
```console
% make testacc TESTARGS='-run=TestAccBatchJobDefinition_updateWithTags' PKG=batch
make: Verifying source code with gofmt...
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go1.23.2 test ./internal/service/batch/... -v -count 1 -parallel 20  -run=TestAccBatchJobDefinition_updateWithTags -timeout 360m
2024/10/21 15:58:11 Initializing Terraform AWS Provider...
=== RUN   TestAccBatchJobDefinition_updateWithTags
=== PAUSE TestAccBatchJobDefinition_updateWithTags
=== CONT  TestAccBatchJobDefinition_updateWithTags
--- PASS: TestAccBatchJobDefinition_updateWithTags (20.33s)
PASS
ok  	github.com/hashicorp/terraform-provider-aws/internal/service/batch	25.533s
```